examples/generate_image_dataset/generate_images.py

```diff
@@ -108,41 +113,16 @@ def single_step(self, action=None, get_observation=False):
                 "set_location": np.array([pose["location_x"], pose["location_y"], pose["location_z"]], np.float64),
                 "set_rotation": np.array([pose["rotation_pitch"], pose["rotation_yaw"], pose["rotation_roll"]], np.float64)})
 
+        observation_components_to_modify = { render_pass: ["camera." + render_pass] for render_pass in config.SIMULATION_CONTROLLER.CAMERA_AGENT.CAMERA.RENDER_PASSES }
```

Comment:
The code this is replacing was inside the `if not args.benchmark` block. Why is this code not inside the same block? I would not want to run this code when benchmarking.

Reply:
Good catch! Updated this.